packages/@aws-cdk/aws-codepipeline/lib/stage.ts

```diff
@@ -29,25 +41,37 @@ export class Stage extends cdk.Construct {
      */
     constructor(parent: Pipeline, name: string) {
```

Comment:
I was expecting the pipeline/stage relationship to also be modeled more idiomatically as a prop
Instead of abusing the construct tree. Let’s align the API completely